app/src/util.tsx

```diff
@@ -63,36 +63,44 @@ export function Copy({
   children,
   clipboardContents,
 }: {
-  children(copy: () => void, copied: boolean): React.ReactElement;
+  children(copy: undefined | (() => void), copied: boolean): React.ReactElement;
   clipboardContents: string;
 }) {
-  const [copied, setCopied] = React.useState(false);
-  const copy = React.useCallback(() => {
-    navigator.clipboard.writeText(clipboardContents).then(() => {
-      setCopied(true);
-    });
-  }, [setCopied, clipboardContents]);
+  if ("clipboard" in navigator) {
+    const [copied, setCopied] = React.useState(false);
```

Comment:
self-review: technically this breaks the "don't call hooks in conditionals" rule, but I don't believe it to be possible for `"clipboard" in navigator` to change within a pageload, so we're always executing one path or the other.